src/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { SortableProperties } from 'ui_framework/services';
+import { Pager } from 'ui/pager';
```

Comment:
We should probably migrate this into the UI Framework. Probably not necessarily as part of this PR though.